Assembler.Base/Releasing/PositionCountValidators/BaseAssemblingPositionCountValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.RawAssemblingEntities;
+using Microsoft.Extensions.Logging;
+
+namespace Assembler.Base.Releasing.PositionCountValidators
+{
+    public abstract class BaseAssemblingPositionCountValidator : IValidator<RawMessageInAssembly>
+    {
+        private readonly int _minimumFramesCount;
+        private readonly ILogger _logger;
+
+        protected BaseAssemblingPositionCountValidator(int minimumFramesCount, ILoggerFactory loggerFactory)
+        {
+            _minimumFramesCount = minimumFramesCount;
+            _logger = loggerFactory.CreateLogger(ToString());
```

Comment:
Why calling ToString? if someone will override it and write "This is the magnificent Validator" you wont want it to appear in your log. just GetType should be good

Reply:
You are right
f